IntegrationTests/UserSettings/Search.settings

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Are these user settings needed? I wonder if we can ignore them

Reply:
we probably can, specially in the `IntegrationTests`